### ISSUE TITLE
Circular call to script handler - composer 1.0.0-alpha11

### ISSUE THREAD

**Issue by mbman** [francoispluchino/composer-asset-plugin#179]
Doesn't work with composer version `1.0.0-alpha11` - throws a `Skipped tag v1.2.3, Circular call to script handler 'fxp.composer.add.vcs.repositories' detected` error. Works fine with earlier composer versions.

**Comment by francoispluchino** [comment]
It's 'normal', because the plugin require the last `dev` version of Composer (see the [release note](https://github.com/francoispluchino/composer-asset-plugin/releases/tag/v1.1.1)).

**Comment by antonmarin** [comment]
Same error. composer 1.1.1. How to fix?

**Comment by francoispluchino** [comment]
Update composer to >=1.2.0 and plugin.

**Comment by antonmarin** [comment]
Nope (( Worked for some, but not all. Cache cleared

**Comment by antonmarin** [comment]
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Composer version 1.2.0 2016-07-19 01:28:52

**Comment by antonmarin** [comment]
worked with fxp/composer-asset-plugin:1.1.4

**Comment by francoispluchino** [comment]
Your config, because I haven't your bug, and I use Composer v1.2.0 with the plugin 1.2.0.

**Comment by antonmarin** [comment]
you mean composer.json?

**Comment by antonmarin** [comment]
```
{
    "autoload": {
      "psr-4": {
        "general\\": "general/",
        "edit\\": "edit/",
        "frontend\\": "frontend/"
      }
    },
    "config": {
        "vendor-dir": "general/data/lib/vendor",
        "sort-packages": true
    },
    "extra": {
        "asset-installer-paths": {
            "npm-asset-library": "general/data/lib/vendor/npm",
            "bower-asset-library": "general/data/lib/vendor/bower"
        }
    },
    "license": "proprietary",
    "require": {
        "php": ">=5.4.0",
        "ext-PDO": "^1.0",
        "ext-intl": "*",
        "ext-openssl": "*",
        "ext-pdo_mysql": "^1.0",
        "bower-asset/font-awesome": "4.2.0",
        "bower-asset/metisMenu": "1.1.3",
        "bower-asset/morrisjs": "0.5.1",
        "creocoder/yii2-nested-sets": "dev-master",
        "doctrine/annotations": "1.2.7",
        "doctrine/common": "2.5.3",
        "fedemotta/yii2-gridstack": "1.1",
        "frostealth/yii2-array-field": "1.0.2",
        "gregwar/captcha": "1.1.1",
        "jms/serializer": "0.16.0",
        "kartik-v/yii2-date-range": "1.6.7",
        "kartik-v/yii2-widget-typeahead": "1.0.1",
        "leandrogehlen/yii2-treegrid": "1.0.1",
        "nizsheanez/yii2-asset-converter": "1.3.0",
        "sirian/yandex-market-language-parser": "3.0.2",
        "symfony/class-loader": "2.6.13",
        "symfony/console": "2.6.13",
        "symfony/css-selector": "2.6.13",
        "symfony/dom-crawler": "2.6.13",
        "symfony/yaml": "2.6.13",
        "tinymce/tinymce": "4.4.1",
        "yiisoft/yii2": "2.0.9",
        "yiisoft/yii2-bootstrap": "2.0.6",
        "yiisoft/yii2-debug": "2.0.6",
        "yiisoft/yii2-gii": "2.0.5",
        "yiisoft/yii2-jui": "2.0.6",
        "yiisoft/yii2-swiftmailer": "2.0.5",
        "zendframework/zend-barcode": "2.3.9",
        "zendframework/zend-soap": "2.3.9",
        "zendframework/zendrest": "2.0.2"
    },
    "require-dev": {
        "ext-pdo_sqlite": "^1.0",
        "codeception/aspect-mock": "^0.5.5",
        "codeception/codeception": "^2.1",
        "codeception/specify": "^0.4.3",
        "phing/phing": "^2.12",
        "yiisoft/yii2-codeception": "^2.0",
        "yiisoft/yii2-faker": "^2.0"
    },
}
```

**Comment by francoispluchino** [comment]
Fixed by d7d189e8ffac26a2e39e26a9c75e1fc5c2e2b98d.

But your exemple has a dependency problem: `raphael` require `eve` with a GIT URL and short SHA key from `v2.2.0`, but Composer is not compatible with the short SHA (see the issue #246).

**Comment by antonmarin** [comment]
Thanks. But why worked with 1.1.4?
